src/CONFIG.js

```diff
@@ -1,51 +1,53 @@
 import _ from 'underscore';
+import lodashGet from 'lodash.get';
 import {Platform} from 'react-native';
 import Config from 'react-native-config';
+import getPlatform from './libs/getPlatform/index';
+
+// Set default values to contributor friendly values to make development work out of the box without an .env file
+let useNgrok = lodashGet(Config, 'USE_NGROK', 'false');
+let expensifyCashURL = lodashGet(Config, 'URL_EXPENSIFY_CASH', 'https://expensify.cash/');
+let expensifyURL = lodashGet(Config, 'EXPENSIFY_URL_COM', 'https://www.expensify.com/');
+const ngrokURL = lodashGet(Config, 'NGROK_URL', null);
+const useWebProxy = lodashGet(Config, 'USE_WEB_PROXY', 'true');
+const expensifyComWithProxy = getPlatform() === 'web' && useWebProxy === 'true' ? '/' : expensifyURL;
 
 // Let's make everyone's life just a bit easier
 // by adding / to the end of any config URL's if it's not already present
-if (_.isString(Config.NGROK_URL) && !Config.NGROK_URL.endsWith('/')) {
-    Config.NGROK_URL += '/';
+if (_.isString(useNgrok) && !useNgrok.endsWith('/')) {
```

Comment:
These should be `ngrokURL` instead of `useNgrok` I believe

Reply:
AH yes, nice catch. Thanks